### ISSUE TITLE
Bridge CL PRO terms to uniprot

### ISSUE THREAD

**Issue by dosumis** [Cellular-Semantics/CL_KG#53]
See https://github.com/obophenotype/cell-ontology/issues/2293

This query bridges some proportion of PRO terms to terms for Mouse and Human UNIPROT CURIEs: https://api.triplydb.com/s/Q1te8-HvE

**Comment by ubyndr** [comment]
Hi @dosumis. What property should we use to bridge these terms?

**Comment by dosumis** [comment]
@dillerm  - Do you have a suggestion for this.  I've been trying but failing to find what to use in https://biolink.github.io/biolink-model/predicates.html

**Comment by dillerm** [comment]
Yes, it [looks like this is how PRO links their species-specific protein terms to species-independent ones](https://ontobee.org/ontology/PR?iri=http://purl.obolibrary.org/obo/PR_000008784).

**Comment by dosumis** [comment]
Agreed we should use SubClassOf and UberGraph query as first step-- then investigate other strategies on linked CL ticket to see if can go further.

**Comment by dosumis** [comment]
Decisions from CL markers meeting 

- [ ] Document how much coverage we get with UberGraph queries (What proportion of PRO terms directly linked to CL terms can we map to uniprot IDs for Mouse and Human).
- Investigate whether we can get better coverage using 
    - [ ] PRO/PIR APIs as document by Darren Natale in ticket.
    - [ ] Translator API to convert from PR to Uniprot
Could we combine all 3?

- [ ] Implement whichever mechanism gives best coverage in CL_KG.
